composer.json

```
@@ -76,13 +76,15 @@
         "liip/imagine-bundle": "^1.9 || ^2.0",
         "matthiasnoback/symfony-dependency-injection-test": "^3.1",
         "nelmio/api-doc-bundle": "^2.11",
+        "sensio/framework-extra-bundle": "^3.0.13 || ^4.0 || ^5.0",
         "sonata-project/admin-bundle": "^3.31",
         "sonata-project/block-bundle": "^3.14",
         "sonata-project/datagrid-bundle": "^2.3",
         "sonata-project/doctrine-orm-admin-bundle": "^3.4",
```

```suggestion
        "sonata-project/doctrine-orm-admin-bundle": "^3.12",
```
This will require doctrine bundle